tests/run.sh

```diff
@@ -200,8 +200,6 @@ export TENANTADM_STRIPE_API_KEY=$STRIPE_API_KEY
 if [ -n "$K8S" ]; then
     export KUBECONFIG="${HOME}/kubeconfig.${K8S}"
     aws eks update-kubeconfig --region $AWS_DEFAULT_REGION --name $AWS_EKS_CLUSTER_NAME --kubeconfig ${HOME}/kubeconfig.${K8S}
-    kubectl config set-context --current --namespace=$K8S
```

Comment:
Q: Not so familiar with k8s. Why remove this?

Reply:
I added `kubernetes` dependency (which uses k8s API underneath) into `tests/requirements/python-requirements.txt`, which is capable of setting default context and performing all necessary actions on staging k8s cluster.